rstudio.Rproj

```diff
@@ -7,6 +7,7 @@ AlwaysSaveHistory: No
 EnableCodeIndexing: Yes
 UseSpacesForTab: Yes
 NumSpacesForTab: 3
+InsertNativePipeOperator: No
```

Comment:
Nit: Can we call this "use native pipe operator"?

Reply:
Also, does this need to live within the `.Rproj` file? We also have the notion of project-specific preferences that _don't_ live in the project file; those are for project-local preferences that don't necessarily need to be "shared" by anyone else working in that project.

Reply:
Nope it doesn't, I just didn't think of that

Reply:
Actually, could you give me an example of a setting like this? I'm seeing several settings where (Default) inheriting the global option is a choice, and we don't write it to the r project file if it isn't set/different from the default, but it looked to me like we always write the option to the file when it differs from the default

Reply:
See also: https://github.com/rstudio/rstudio/issues/7135

Reply:
Some of the options in the Build Tools are an example here; for example, in Roxygen Options, for "Automatically roxygenize when running:":

<img width="378" alt="Screen Shot 2022-03-22 at 10 54 08 AM" src="https://user-images.githubusercontent.com/1976582/159544670-638461f9-4e57-4ff6-94ea-53c364050398.png">

For example, the "Install and Restart" preference, which gets set here:

https://github.com/rstudio/rstudio/blob/56597a1893fa5dabcf9aae4c899f38eb39413e83/src/gwt/src/org/rstudio/studio/client/projects/ui/prefs/buildtools/BuildToolsRoxygenOptionsDialog.java#L53

Which is eventually set as part of this, in `auto_roxygenize_options`:

https://github.com/rstudio/rstudio/blob/56597a1893fa5dabcf9aae4c899f38eb39413e83/src/cpp/session/projects/SessionProjects.cpp#L422-L441

And so those are read from a separate project-specific options file:

https://github.com/rstudio/rstudio/blob/56597a1893fa5dabcf9aae4c899f38eb39413e83/src/cpp/session/projects/SessionProjectContext.cpp#L875-L878

And an example where that exists in my local `renv` project:

```
kevin@MBP-P2MQ:~/r/pkg/renv/.Rproj.user [main]
$ find . -name "build_options"
./142DE28A/build_options
./2FA46A95/build_options
```

See also:

https://github.com/rstudio/rstudio/blob/56597a1893fa5dabcf9aae4c899f38eb39413e83/src/cpp/session/projects/SessionProjects.cpp#L517-L541

So basically, project options which might want to be "shared" by any user who is working in that project will become part of the project file, but "user-specific project options" might use their own separate project-local storage.

Reply:
Ah I see. It looks like the VCS options are another example that have their own separate storage, rather than being written to the RProj file. I started down this route, but because there's not an existing appropriate project-local place to write this to (neither build options nor vcs options would be appropriate) I'd have to create my own, and having started doing so it seems like a lot more work than just writing it to the RProj file. 

Is there a significant disadvantage to writing it to the RProj file? While there's not a strict need to share among collaborators on a project, having an inconsistent mix of the different pipe types throughout the code doesn't necessarily seem ideal either (just as you'd prefer not to have a mix of different tab widths between files for collaborators on a project)

Reply:
In this case, I think it might be fine. It's unlikely that multiple collaborators on the same project would want to use different default pipe styles in one project.